repose-aggregator/artifacts/filter-bundle/src/config/filters/merge-header.cfg.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<merge-header xmlns='http://docs.openrepose.org/repose/merge-header/v1.0'>
+    <!-- both response/request configs are optional, if neither are specified, this filter is a no-op -->
+    <response>
+        <header>your-mother</header>
+        <header>your-father</header>
+        <header>your-sister</header>
+    </response>
```

Comment:
And I honestly didn't look at these for content after I decided to use the already tested examples from the filter META-INF directories.